Services/WebContent/CMakeLists.txt

```diff
@@ -39,15 +39,16 @@ if (ENABLE_QT)
     target_compile_definitions(WebContent PRIVATE HAVE_QT=1)
 
     if (NOT DEFINED LADYBIRD_AUDIO_BACKEND)
```

Comment:
I changed this logic to have QtMultimedia as a 'required fallback' on master. ... it should probably just be a 'real option'. in the audio.cmake helper file.

Reply:
For now I just decided to drop that change and also build qtmultimedia from vcpkg (although it doesn't find any audio backends)